spec/classes/software_social_skype_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'spec_helper'
+
+describe 'software::social::skype' do
+  on_supported_os.each do |os, facts|
+    context "on #{os}" do
+      let(:facts) do
+        facts
+      end
+
+      context 'with defaults' do
+        if facts[:operatingsystem] == 'Darwin'
+          it { is_expected.to compile.with_all_deps }
+          it { is_expected.to contain_package(/Skype-.+/).with_provider('appdmg') }
+        elsif facts[:operatingsystem] == 'Ubuntu'
+          it { is_expected.to compile.with_all_deps }
+          it {
+            is_expected.to contain_apt_source('skype-partner')
```

Comment:
As before (now shown as "outdated" in the PR), there's clearly some syntax issue with `apt_source` here. Not sure though, why the [compiling into a catalogue](https://travis-ci.org/edestecd/puppet-software/builds/290355530#L1123-L1125) is failing.

@edestecd Can you suggest how to reference `apt::source` with `contains_`?

Reply:
I think you do this: 
```ruby
# replace :: with __ in resource names
is_expected.to contain_apt__source('skype-partner')
```